src/libs/OptionsListUtils.ts

```diff
@@ -2303,11 +2346,16 @@ function filterOptions(options: Options, searchInputValue: string): Options {
     }, options);
 
     const recentReports = matchResults.recentReports.concat(matchResults.personalDetails);
+    const userToInvite = getUserToInviteOption({
+        searchValue,
+        betas,
+        shouldGetOptionToInvite: recentReports.length === 0,
```

Comment:
Methods usually should not have "canceling" arguments (maybe there are some extreme exceptions). But it looks like we can just set `userToInvite` to `null`? 

It's like having a method called `eatFood(doNotEatFood)` (can you tell it's lunch time for me?) 😄

Reply:
> Methods usually should not have "canceling" arguments (maybe there are some extreme exceptions). But it looks like we can just set userToInvite to null?

Yes we can just set `userToInvite` to `null`.